### ISSUE TITLE
Documentation Reorganization

### ISSUE THREAD

**Issue by endigo9740** [Brain-Bones/skeleton#237]
Prerequisite: https://github.com/Brain-Bones/skeleton/pull/233

Let's adjust a few things:

- [x] Move Tailwind install from "Getting Started > Install" to "Tailwind Settings"
- [X] Break up the "Getting Started > Styling > Preview" page into "Tailwind Elements > X" (Core and Typography separated)
- [X] Update the new "Tailwind Elements > Forms" section to match the component page layout flow (add sandbox?)
- [x] Rename the "Getting Started > Usage with Astro" to "(App?) Scaffold" and make this a tabbed section for SvelteKit/Vite/Astro

**Comment by endigo9740** [comment]
Note, a couple of these items were completed when #233 was merged into the dev branch. I've updated the list accordingly.
